spec/backupsss/tar_spec.rb

```diff
@@ -63,6 +63,22 @@
     end
   end
 
+  describe '#tar_command' do
+    context 'compress_archive true' do
+      subject { Backupsss::Tar.new('some/src', 'some/dest.tar').tar_command }
+
+      it { is_expected.to eq('tar -zcvf') }
+    end
+
+    context 'compress_archive false' do
+      subject do
```

Comment:
i had to do end there because of Rubocop :cry: Should i change them both to 'do..end' to match?